### PR TITLE
Add `try-runtime` to Rococo runtime

### PR DESCRIPTION
**Pull request by tdimitrov**

### CODE REVIEW

**Review by alexgparity — APPROVED** [review]
LGTM

File: runtime/rococo/src/lib.rs [hunk 1]
```diff
@@ -2043,3 +2063,38 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(all(test, feature = "try-runtime"))]
+mod remote_tests {
+	use super::*;
+	use frame_try_runtime::runtime_decl_for_TryRuntime::TryRuntime;
+	use remote_externalities::{
```
Comment (ggwpez) [inline runtime/rococo/src/lib.rs:2071]: I cannot find `remote-externalities` in the `Cargo.toml` of Rococo.  
So this should not compile. The CI is not checking that as it seems :man_facepalming:

Comment (ggwpez) [reply runtime/rococo/src/lib.rs:2071]: You can probably ignore this. Will fix it in a follow up.

Comment (tdimitrov) [reply runtime/rococo/src/lib.rs:2071]: How exactly should this test be executed? `cargo test --features try-runtime` compiles just fine, but I agree it should fail.

**Comment by tdimitrov** [comment]
> For the future: please dont force-push, as mentioned in [CONTRIBUTING](https://github.com/paritytech/substrate/blob/master/docs/CONTRIBUTING.adoc#rules).

Will do, sorry about that.

**Comment by tdimitrov** [comment]
bot merge

**Pull request merged**